DNN Platform/Library/Security/PortalSecurity.cs

```diff
@@ -1018,6 +1029,15 @@ private static string FormatRemoveSQL(string strSQL)
             return BadStatementRegex.Replace(strSQL, " ").Replace("'", "''");
         }
 
+        /// <summary> This function removes control characters from the string.</summary>
+        /// <param name="str">This is the string to be filtered.</param>
+        /// <returns>Filtered UserInput.</returns>
+        /// <remarks>This is a private function that is used internally by the <see cref="InputFilter"/> function.</remarks>
+        private static string FormatRemoveControlCharacters(string str)
+        {
+            return ControlCharactersRegex.Replace(str.Replace('\t', ' '), string.Empty);

```

Comment:
Should removing control characters replace a tab with a space?

Reply:
I wasn't sure about this one, since a tab is still some whitespace. I didn't know if you still want to see in the logging that a whitespace/tab was used.

If you want I can replace it with an empty string as well.

Reply:
I hadn't realized that [control characters](https://en.wikipedia.org/wiki/Unicode_control_characters) includes tabs and line feeds. Replacing with a space makes sense to me.

Reply:
Should I replace `\n` and `\r` (new lines) with spaces as well?

Reply:
👍🏻 that would make sense to me